modules/xmpp/ChatRoom.js

```diff
@@ -812,6 +839,13 @@ export default class ChatRoom extends Listenable {
         const membersKeys = Object.keys(this.members);
 
         if (!isSelfPresence) {
+            if (this.members[from].liveStreamViewURL) {
```

Comment:
@bbaldino, do you think I should be defensive here instead of assuming the member exists?